config/mutable.go

```diff
@@ -0,0 +1,97 @@
+// Licensed under the Apache License, Version 2.0
+// Details: https://raw.githubusercontent.com/maniksurtani/quotaservice/master/LICENSE
+
+package config
+
+import (
+	"errors"
+
+	pbconfig "github.com/maniksurtani/quotaservice/protos/config"
+)
+
+func CreateBucket(clonedCfg *pbconfig.ServiceConfig, namespace string, b *pbconfig.BucketConfig) error {
+	if namespace == GlobalNamespace {
+		// TODO(steved) what to do?
```

Comment:
Right, should've been clearer. If there is already a global default bucket config, should this fail? Should we just allow update and delete for globals/defaults?